Framework/Kernel/src/Unit.cpp

```diff
@@ -583,6 +583,54 @@ double dSpacing::conversionTOFMax() const { return DBL_MAX / factorTo; }
 
 Unit *dSpacing::clone() const { return new dSpacing(*this); }
 
+// ==================================================================================================
+/* D-SPACING Perpendicular
+ * ==================================================================================================
+ *
+ * Conversion uses equation: d^2 = lambda^2 - 2[Angstrom^2]*ln(sin(theta))
```

Comment:
Some more information may be found here, in the talk of A. Houben: https://indico.esss.lu.se/event/785/session/5/material/0/3.pdf